### ISSUE TITLE
angular-whitespace is deprecated

### ISSUE THREAD

**Issue by huineng** [mgechev/tslint-angular#13]
i get this warning now (angular 7.2.15)

```
angular-whitespace is deprecated. Use a formatter like Prettier for formatting purposes
```

thanks

**Comment by mgechev** [comment]
I'll update and publish a new release today.

**Comment by huineng** [comment]
perfect response !

**Comment by mgechev** [comment]
2.0.3 is out.
